src/app/counter/counter.component.ts

```diff
@@ -12,4 +12,18 @@ export class CounterComponent implements OnInit {
   ngOnInit(): void {
   }
 
+  countForm = new FormGroup({
+    countBox: new FormControl(0)
+  });
+
+  onclick(){
+    this.countForm.setValue({
+      countBox: this.countForm.value.countBox + 1
+    })
+  }
+  reset(){
+    this.countForm.setValue({
+      countBox: this.countForm.value.countBox = 0
```

Comment:
I hope you are aware of when we use patchValue and setValue.
Since here you have only one formControl element so either of the two can be used.
For updating only fewer formControls we use patchValue(). Whereas if you want to use setValue(), you need to mention all the formControls which you introduced while initiating the formGroup.

Also, I doubt your procedure for setting the value.
I think it should be directly set as 
this.countForm.setValue({
      countBox: 0
})